src/Common/src/TypeSystem/Common/DefType.FieldLayout.cs

```diff
@@ -110,5 +110,4 @@ public int InstanceByteCount
             get;
         }
     }
-
 }
```

Comment:
Yes - if you see files with missing new at the end of the file (where the github diff tool displays the little red mark). If there is too many of them, make a separate commit for it.